packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/AddAccountModal/AddAccountModal.tsx

```diff
@@ -8,10 +8,10 @@ import {
 } from '@suite-common/wallet-core';
 import { arrayPartition } from '@trezor/utils';
 import { networks, Network, NetworkSymbol, NetworkAccount } from '@suite-common/wallet-config';
-import { CollapsibleBox, NewModal } from '@trezor/components';
+import { CollapsibleBox, NewModal, Tooltip } from '@trezor/components';
 import { FirmwareType } from '@trezor/connect';
 import { spacings, spacingsPx } from '@trezor/theme';
-import { Translation, CoinList, TooltipSymbol } from 'src/components/suite';
+import { Translation, CoinList } from 'src/components/suite';
```

Comment:
It's still used here, TS gives error.
```suggestion
import { Translation, CoinList, TooltipSymbol } from 'src/components/suite';
```